FrequencyConversion/FC_solve_EoM.py

```diff
@@ -0,0 +1,237 @@
+"""Functions to calculate the numerical solution to the FC equations of motion using linear algebra"""
+
+import numpy as np
+from scipy.linalg import expm
+from scipy.special import hermite
+
+# pylint: disable=invalid-name
+# pylint: disable=too-many-arguments
+# pylint: disable=too-many-locals
+# pylint: disable=consider-using-enumerate
+
+
+def dag(M):
+    """
+    Gives the Hermitian conjugate of input matrix.
+
+    Args:
+        M (array): input matrix
+
+    Returns:
+        array: the Hermitian conjugate matrix
+    """
+    return M.conj().T
+
+
+def pulse_shape(dw, sig, n=0):
+    """
+    Gives a Hermite-Gaussian pulse shape.
+
+    Args:
+        dw (array): frequency vector or grid
+        sig (float): width of the pulse
+        n (int): index of Hermite polynomial used. Default is 0 (first Hermite polynomial)
+
+    Returns:
+        array: pulse shape
+    """
+    gaussian = np.exp(-(dw**2) / (2 * sig**2)) / np.sqrt(2 * np.pi * sig**2)
```

Comment:
This seems to be L1 normalization, whereas we want to use L2 normalization.  Denominator should be (pi*sig**2)^(1/4)